pkg/template/generated/listers/template/v1/template_expansion.go

```diff
@@ -2,26 +2,27 @@ package v1
 
 import (
 	templateapi "github.com/openshift/origin/pkg/template/apis/template"
```

Comment:
This dependency is illegal.  I'm try to push through, but some debt is going to fall out.